app/assets/stylesheets/publisher.scss

```diff
@@ -243,14 +237,6 @@
     .hide-location { display: inline-block; }
     .locator { display: none; }
   }
-
-  .twitter-typeahead {
-    left: -1px;
-    // Override inline rule of Typeahead
-    // scss-lint:disable ImportantRule
-    position: absolute !important;
```

Comment:
I'm unable to reproduce your issues with the width of the comment mentions, but maybe there is just some misunderstanding. This is the diff between your current version and the one I'm talking about:

```diff
diff --git a/app/assets/stylesheets/publisher.scss b/app/assets/stylesheets/publisher.scss
index 70e89b3d0..f95480c94 100644
--- a/app/assets/stylesheets/publisher.scss
+++ b/app/assets/stylesheets/publisher.scss
@@ -237,6 +237,13 @@
     .hide-location { display: inline-block; }
     .locator { display: none; }
   }
+
+  .twitter-typeahead {
+    // Override inline rule of Typeahead
+    // scss-lint:disable ImportantRule
+    position: absolute !important;
+    // scss-lint:enable ImportantRule
+  }
 }
 
 .publisher-buttonbar {
```

This only adds styles to the publisher (inside `.publisher`), so I don't see how that could break the comment mentions styles.

The problem with the current version is the fact that mention suggestions are between the textarea and photos / location / polls / the submit button. This means that when you start mentioning someone everything below the suggestions might move with every single character you type.